driver-core/pom.xml

```diff
@@ -201,7 +201,7 @@
                         <_include>-osgi.bnd</_include>
                         <Import-Package>
                             <!-- JNR does not provide OSGi bundles, so exclude it; the driver can live without it -->
-                            <![CDATA[com.google.common*;version="[14.0,20)",!jnr.*,*]]></Import-Package>
+                            <![CDATA[com.google.common*;version="[14.0,20)",!jnr.*,!sun.misc.*,*]]></Import-Package>
```

Comment:
With the current code, `sun.misc.Unsafe` is referenced from `LongAdder`, which makes it a non-optional dependency from OSGi's standpoint. So imho we shouldn't exclude it from imported packages but rather add `sun.misc` to the list of system packages (i.e. packages provided by the system bundle). That would mean the following change in `BundleOptions`:

```java
systemPackages("sun.misc", "org.testng", "org.junit", ...)
```

That said, I think we should  instead make that dependency truly optional by providing an alternative implementation. Then we could mark it as an optional package import.